policy.md

```diff
@@ -226,24 +313,63 @@ Each audit statement MUST be accompanied by documentation of the audit team qual
+- the full and complete CRL URL MUST be empty.
 
-Some Store policies MAY require the completion of an annual [self-assessment](https://www.ccadb.org/cas/self-assessment) using the CCADB template at various times to evaluate the conformance of the CA Owner’s policies against industry policies and practices.
+Under normal operating conditions, the CRL URLs provided by CAs in accordance with this section MUST be available such that relying parties are able to successfully retrieve the current CRL every 4 hours.
 
-If an annual CCADB self-assessment is required by the individual Store policy, a single self-assessment MAY cover multiple CAs operating under both the same CP and CPS(s), or combined CP/CPS. CAs not operated under the same CP and CPS(s) or combined CP/CPS MUST be covered in a separate self-assessment.
+### 6.3 Cross-certification across PKI Hierarchies
 
-The self-assessment submission date is determined by the earliest appearing "BR Audit Period End Date" field specified in any of the CA Owner’s "CA Owner/Certificate" CCADB root records that are included in one or more Stores. 
+Some Root Store Operators participating in the CCADB maintain policy requirements such that hierarchies included in the corresponding root store(s) are dedicated to a specific PKI use case, for example only supporting TLS server authentication. 
 
-An annual self-assessment MUST be completed and submitted to the CCADB within 92 calendar days from the CA Owner's earliest appearing root record "BR Audit Period End Date" that is after December 31, 2023. CA Owners SHOULD submit the self-assessment to the CCADB at the same time as uploading audit reports.
+In support of ubiquity use-cases, CA Owners sometimes issue cross-certificates from non-dedicated hierarchy Root CAs (i.e., “multi-purpose hierarchies”) to dedicated-use hierarchy Root CAs. 
 
-CA Owners SHOULD always use the latest available version of the self-assessment template. CA Owners MUST NOT use a version of the self-assessment template that has been superseded by more than 90 calendar days before their submission.
+Beginning June 15, 2025, when new cross-certificates are issued across PKI hierarchies and the Subject CA’s public key (a) exists in or (b) might exist in a publicly-trusted self-signed Root CA certificate whose hierarchy should be considered dedicated to a specific PKI use case, the following EKU values MUST exist:
 
-## 6. Mailshots ##
+| If the subject hierarchy is dedicated to… | EKU Presence? | EKU OID Permitted? |
+|---|---|---|
+| TLS server authentication | MUST | Only 1.3.6.1.5.5.7.3.1                       |
+| TLS client authentication | MUST | Only 1.3.6.1.5.5.7.3.2                       |
+| TLS (generic)             | MUST | Only 1.3.6.1.5.5.7.3.1 and 1.3.6.1.5.5.7.3.2 |
```

Comment:
This seems to allow for TLS server auth and TLS client auth still combined. Should the same allowance be specified for S/MIME as well, since the SBRs allow both?